src/kinetics/Reaction.cpp

```diff
@@ -256,6 +256,11 @@ void Reaction::setRate(shared_ptr<ReactionRate> rate)
         reactants.erase("(+M)");
         products.erase("(+M)");
     }
+
+    if (reactants.count("M") && std::dynamic_pointer_cast<PlogRate>(m_rate)) {
+        throw InputFileError("Reaction::setRate", input, "Invalid Syntax: found "
+            "superfluous 'M' in pressure-dependent-Arrhenius reaction.");
```

Comment:
This is a bit of a nitpick, but I would argue that this is not a syntax error -- this is still a parsable reaction expression. I'd suggest just starting the error message with "Found superfluous...".